docs/how-to-review-new-components.md

```diff
@@ -0,0 +1,7 @@
+
+## How-to review new components
+
+Each pull request will be automatically deploy on [Zeit](https://zeit.co) via `now`,
+producting a unique URL such as https://flamingo-git-gyndav-featzeit.heetch-eng.now.sh
```

Comment:
How is this `flamingo-git-gyndav-featzeit` built?
Is this related to the branch's name?

Reply:
What do you mean? I'm afraid I don't really understand the underlying question there.
If you meant the name, indeed, sounds like `project-name-platform-branch_name`. Yet another alias, that we can prevent. The original one here was https://flamingo-fw5vr9suf.now.sh/

Reply:
Yeah sorry I was talking the naming of this URL
`project-name-platform-branch_name` sounds good 👍